main/webapp/modules/core/langs/translation-en.json

```
@@ -244,6 +244,7 @@
     "core-dialogs/sql-exporter": "SQL Exporter",
     "core-dialogs/custom-tab-exp": "Custom Tabular Exporter",
     "core-dialogs/select-columns-dialog": "Select columns",
+    "core-dialogs/unstar-expression": "Unstar Expression?",
```

```suggestion
    "core-dialogs/unstar-expression": "Unstar expression?",
```
We are in the process of normalizing the case of all text in the UI (#3363), so let's get this one right from the start.